src/components/ItemCard.css

```diff
@@ -0,0 +1,15 @@
+.Item-card {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  background-color: var(--color-items-background);
+  margin: 1rem;
+  border: hidden;
+  border-radius: 15px;
+}
+
+.frame {
+  border: var(--color-icons) solid;
+}
```

Comment:
why is this an extra class and not in the .Item-card class?

Reply:
I guess I was trying something out.